src/main/java/com/questhelper/helpers/quests/heroesquest/HeroesQuest.java

```diff
@@ -616,14 +616,9 @@ public List<Requirement> getGeneralRequirements()
 		req.add(new SkillRequirement(Skill.FISHING, 53, true));
 		req.add(new SkillRequirement(Skill.HERBLORE, 25, true));
 		req.add(new SkillRequirement(Skill.MINING, 50, true));
-		if (isInBlackArmGang)
-		{
-			req.add(new QuestRequirement(QuestHelperQuest.SHIELD_OF_ARRAV_BLACK_ARM_GANG, QuestState.FINISHED));
-		}
-		else
-		{
-			req.add(new QuestRequirement(QuestHelperQuest.SHIELD_OF_ARRAV_PHOENIX_GANG, QuestState.FINISHED));
-		}
+		Conditions shieldOfArravRequirement = new Conditions(new Conditions(new QuestRequirement(QuestHelperQuest.SHIELD_OF_ARRAV_BLACK_ARM_GANG, QuestState.FINISHED)));
+		shieldOfArravRequirement.setText("Finished Shield of Arrav");
```

Comment:
This is very wrapped in Conditions, could it just be a `new QuestRequirement(..., QuestState.FINISHED, "Finished Shield of Arrav")?